### PR TITLE
graphql: Instrument more fields

### PR DESCRIPTION
**Pull request by ryanslade**
Add the following fields:

* Duration in milliseconds since honeycomb has built in support for
this.
* Whether there was an error as honeycomb has support for tagging
requests as errored.
* Error calculating cost
* Calculation error
* Request name
* Request source

### CODE REVIEW

File: cmd/frontend/graphqlbackend/graphqlbackend.go [hunk 1]
```diff
@@ -83,17 +77,32 @@ func (h honeycombTracer) TraceQuery(ctx context.Context, queryString string, ope
...
 		ev.AddField("operationName", operationName)
 		ev.AddField("isInternal", sgtrace.IsInternalRequest(ctx))
 		d := time.Since(start)
 		ev.AddField("durationMicroseconds", d.Microseconds()) // Deprecated
-		ev.AddField("durationSeconds", d.Seconds())
+		ev.AddField("durationSeconds", d.Seconds())           // Deprecated
+		// Honeycomb has built in support for latency if you use milliseconds. We
+		// multiply seconds by 1000 here instead of using d.Milliseconds() so that we
+		// don't truncate durations of less than 1 millisecond.
+		ev.AddField("durationMilliseconds", d.Seconds()*1000)
```
Comment (tsenart) [inline cmd/frontend/graphqlbackend/graphqlbackend.go:91]:
```suggestion
		ev.AddField("durationMilliseconds", d.Milliseconds())
```

**Review by tsenart — APPROVED** [review]

File: cmd/frontend/graphqlbackend/graphqlbackend.go (hunk 1 above)
Comment (ryanslade) [reply cmd/frontend/graphqlbackend/graphqlbackend.go:91]: See the comment above 😄

**Comment by ryanslade** [comment]
Add the following fields:

* Duration in milliseconds since honeycomb has built in support for
this.
* Whether there was a query error as honeycomb has support for tagging
requests as errored.
* If there was an error calculating cost
* The calculation error
* Request name
* Request source

File: cmd/frontend/graphqlbackend/graphqlbackend.go (hunk 1 above)
Comment (tsenart) [reply cmd/frontend/graphqlbackend/graphqlbackend.go:91]: 🤦🏼

**Pull request merged**